src/components/chats/index.js

```diff
@@ -0,0 +1,71 @@
+import useTranslation from 'hooks/useTranslation';
+import { useEffect, useState } from 'react';
+import { useGetConversationQuery } from 'services/conversation/conversation';
+import profile from 'assets/profile.svg';
+import { v4 as uuid } from 'uuid';
+
+import './styles.scss';
+
+const Chats = () => {
+  const [tabSelected, setTabSelected] = useState('HOME');
+  const user = JSON.parse(localStorage.getItem('user'));
+  const { data: matchConversations } = useGetConversationQuery();
+  const [matchList, setMatchList] = useState([]);
```

Comment:
This a personal opinion (and you could disagree!) but I usually write all of the states on the same "block" of code. It makes everything more readable
```suggestion
  const [matchList, setMatchList] = useState([]);
  const [tabSelected, setTabSelected] = useState('HOME');
  
  const { data: matchConversations } = useGetConversationQuery();
  const user = JSON.parse(localStorage.getItem('user'));
```